src/app/fdctl/config.c

```diff
@@ -1125,6 +1124,28 @@ config_parse( int *      pargc,
   if( FD_UNLIKELY( getgid() != 0 && config->gid != getgid() ) )
     FD_LOG_ERR(( "running as gid %i, but config specifies gid %i", getgid(), config->gid ));
 
+  const char * shmem_path = fd_env_strip_cmdline_cstr( pargc, pargv, "--shmem-path","FD_SHMEM_PATH", NULL );
+  if( FD_UNLIKELY( shmem_path ) ) {
+    strncpy( config->hugetlbfs.mount_path, shmem_path, sizeof( config->hugetlbfs.mount_path )-1 );
+  }
+  ulong len = strlen( config->hugetlbfs.mount_path );
```

Comment:
This is a bit error prone and we don't need to do it, just append `/.gigantic` like below.